src/components/Card.js

```diff
@@ -1,59 +1,84 @@
 export default class Card {
-  constructor({ title, link }, cardSelector, handleImageClick) {
-    this._title = title;
+  constructor({
```

Comment:
Needs correction: I can't open the preview `Card.js:33 Uncaught TypeError: this._handleImageClick is not a function
    at HTMLImageElement.<anonymous> (Card.js:33:12)`

Reply:
The preview now opens, sorry I missed that before submitting!